### ISSUE TITLE
Flashbots Exploit

### ISSUE THREAD

**Issue by Primata** [erc721r/ERC721R#8]
Believe Flashbots are still able to exploit since they interact as EOA.

**Comment by RogerPodacter** [comment]
You are correct! I have updated the README accordingly. On the one hand people really shouldn't ever use ERC721R! And yet it has been used to amusing effect many times. It is a true paradox!
